pkg/addons/default/assets/coredns-1.11.json

```diff
@@ -0,0 +1,311 @@
+{
+  "apiVersion": "v1",
+  "items": [
+    {
+      "apiVersion": "v1",
+      "kind": "Service",
+      "metadata": {
+        "annotations": {
+          "prometheus.io/port": "9153",
+          "prometheus.io/scrape": "true"
+        },
+        "labels": {
+          "eks.amazonaws.com/component": "kube-dns",
+          "k8s-app": "kube-dns",
+          "kubernetes.io/cluster-service": "true",
+          "kubernetes.io/name": "CoreDNS"
+        },
+        "name": "kube-dns",
+        "namespace": "kube-system"
+      },
+      "spec": {
+        "clusterIP": "10.100.0.10",
```

Comment:
is this always the same?

Reply:
No, and we handle here: https://github.com/weaveworks/eksctl/pull/878/files#diff-9db8b5b22bd493bd608250a68252142fR211